fullcalendar-app/src/service/EventService.js

```diff
@@ -86,6 +88,28 @@ export default class EventService {
     return window.drupalSettings.fullCalendar?.branch_id || null;
   }
 
+  /**
+   * Retrieves the 'editable' setting from the window.drupalSettings object.
+   * If the 'editable' setting is not defined within the drupalSettings,
+   * it will return TRUE as the default value.
+   *
+   * @returns {boolean} The 'editable' setting value if available, otherwise TRUE.
+   */
+  isEditable() {
+    return window.drupalSettings.fullCalendar?.editable ?? true;
```

Comment:
@froboy is it possible to pass this variable to the application on the run instead of using global window object?

Reply:
@anpolimus possibly? All of the other data was being passed through `drupalSettings` so I did the same here, but we could set options on data attributes or something instead?

Reply:
@froboy nevermind. This application is not build this way.
Usually, when we do app in YUSA we try to put input variables like here:
https://github.com/YCloudYUSA/yusaopeny_activity_finder/blob/5.x/openy_af4_vue_app/src/App.vue#L5
Since this app is not built using this approach this PR is not good time for that.

Reply:
Agreed. We can consider refactoring at another point.